src/pybind/mgr/dashboard/services/ceph_service.py

```diff
@@ -114,14 +106,20 @@ def get_pool_list_with_stats(cls, application=None):
 
             def get_rate(series):
                 if len(series) >= 2:
-                    return differentiate(*list(series)[-2:])
+                    return derivative(*list(series)[-2:], True)
```

Comment:
Same here?

Reply:
Here we already expect a list with 2 or more elements.